modules/tts/tts-adapters/tts-adapter-cereproc/src/main/java/org/daisy/pipeline/tts/cereproc/impl/CereProcEngine.java

```diff
@@ -45,8 +59,21 @@
 
 	private AudioFormat audioFormat;
 	private final int priority;
+	static final Map<String, CereprocTTSUtil> mTtsUtils = new HashMap<String , CereprocTTSUtil>() {
+		{
+			put("sv", new CereprocTTSUtil(Optional.of(new Locale("sv"))));
+			put("en", new CereprocTTSUtil(Optional.of(new Locale("en"))));
+		}
+	};
+
 	private final String[] cmd;
 	private final int expectedMillisecPerWord;
+	private HashMap<Configuration, ThreadLocal <ThreadUnsafeXslTransformer>> mTransformer = new HashMap<>();
+	private URL ssmLxslTransformerURL;
+
+	public void setStyleSheet(URL ssmLxslTransformerURL) {
```

Comment:
Would personally add this to constructor so you don't need mutability.

Reply:
Yes changed it.